packages/snowflake/src/lib/Snowflake.ts

```diff
@@ -0,0 +1,86 @@
+let INCREMENT = 0n;
```

Comment:
Make this a hard private per class instance maybe? That way, different epochs won't use the same increment

Reply:
... hard private static...?

Reply:
No. just a `#increment`

Reply:
Ohhh, wait, duhh, I forgot to say this (welcome to dumb brain)

My idea was you create a Snowflake class with the epoch ONCE, then run, say .deconstruct / .generate on the instance, so no statics

Reply:
gdi Vlad :|

Reply:
SORRY 😢